config/jobs/etcd/etcd-benchmarks-periodic.yaml

```diff
@@ -34,3 +34,6 @@ periodics:
             memory: "8Gi"
     nodeSelector:
       kubernetes.io/arch: amd64
+  tags:
+    - "perfDashPrefix: etcd"
```

Comment:
Went on to set the above prefixes, open to rename to other suitable names.
Sample: 
<img width="1432" height="891" alt="image" src="https://github.com/user-attachments/assets/2474bd30-cd17-40f7-a396-6115d0c7ef88" />